sci-libs/rocBLAS/rocBLAS-4.3.0.ebuild

```diff
@@ -97,7 +99,6 @@ src_configure() {
 		-DBUILD_CLIENTS_TESTS=$(usex test ON OFF)
 		-DBUILD_CLIENTS_BENCHMARKS=$(usex benchmark ON OFF)
 		${AMDGPU_TARGETS+-DAMDGPU_TARGETS="${AMDGPU_TARGETS}"}
-		-D__skip_rocmclang="ON" ## fix cmake-3.21 configuration issue caused by officialy support programming language "HIP"
```

Comment:
Depend on a newer CMake version?

Reply:
only cmake-3.21.1 and cmake-3.21.2 need this, version below this or higher than this don't need or recognize this flag, and will throw a warning (like https://bugs.gentoo.org/829326 described). The two version does not exists in portage tree now.